AmplifyPlugins/API/Tests/AWSAPIPluginTests/AppSyncRealTimeClient/AppSyncRealTimeRequestAuthTests.swift

```diff
@@ -0,0 +1,215 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+import XCTest
+@testable import AWSAPIPlugin
+
+let host = UUID().uuidString
+let apiKey = UUID().uuidString
+let date = UUID().uuidString
+let id = UUID().uuidString
+let data = UUID().uuidString
+let token = UUID().uuidString
```

Comment:
are these used elsewhere? if not, prefer scoping to where they are used, under AppSyncRealTimeRequestAuthTests ?

Reply:
will move these inside Test class.